account-decoder/src/parse_token.rs

```diff
@@ -1,10 +1,16 @@
 use crate::parse_account_data::{ParsableAccount, ParseAccountError};
-use spl_sdk::pubkey::Pubkey;
+use solana_sdk::pubkey::Pubkey;
+use spl_sdk::pubkey::Pubkey as SplPubkey;
 use spl_token::{
     option::COption,
     state::{Account, Mint, Multisig, State},
 };
-use std::mem::size_of;
+use std::{mem::size_of, str::FromStr};
+
+// A helper function to convert spl_token::id() as spl_sdk::pubkey::Pubkey to solana_sdk::pubkey::Pubkey
+pub fn spl_token_id_v1_0() -> Pubkey {
+    Pubkey::from_str(&spl_token::id().to_string()).unwrap()
```

Comment:
Seems like `spl_token` be namespaced as `spl_token_v1.0` too?  That's an existing import so maybe as a follow-up?  That'll set us up for the inevitable v1.1 when it arrives

Reply:
Good call. I think that's fair to roll in to the first commit of this PR.